### PR TITLE
Add .gitattributes

### PR DESCRIPTION
**Pull request by thePanz**

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1,7 @@
+/.coveralls.yml export-ignore
+/.gitattributes export-ignore
+/.gitignore export-ignore
+/.travis.yml export-ignore
+/build.xml
+/phpunit.xml.dist
```
Comment (MalteWunsch) [inline .gitattributes]: Hi @thePanz and thank you for your PR. 

As I'm not experienced in gitattributes, I wonder what the two "path without attributes" lines will result in?

File: .gitattributes [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+/Tests export-ignore
```
Comment (MalteWunsch) [inline .gitattributes]: Is it best practise to exclude the tests from archives? I'm not sure about that.

File: .gitattributes (hunk 1 above)
Comment (thePanz) [reply .gitattributes]: Sorry, the `export-ignore` parts are required, added!

File: .gitattributes (hunk 2 above)
Comment (thePanz) [reply .gitattributes]: IMO the "archived" version of a bundle is the one that is going to be installed on prod environments via composer.
In that case the tests are not needed on "end-users" environments, as those devs are not focused on writing new tests for the library.
Tests will be available when cloning the repository, operation performed when willing to contribute to the project.

WDYT?

Comment (mpdude) [reply .gitattributes]: For example, github.com/symfony/symfony does not exclude them. https://github.com/doctrine/common/blob/master/.gitattributes does.

I would not bother setting up `.gitattributes` myself, but I'm also fine merging this.

**Comment by thePanz** [comment]
Thanks!

**Pull request merged**